### PR TITLE
Update cargo.lock and add vscode to gitignore

### PR DESCRIPTION
**Pull request by zyfjeff**
## Relevant Issue (if applicable)
_If there are Issues related to this PullRequest, please list it._

## Details
1.  Update cargo.lock to latest
2. Add vscode to gitignore for all contrib subdir

## Types of changes

_What types of changes does your PullRequest introduce? Put an `x` in all the boxes that apply:_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Go over all the following points, and put an `x` in all the boxes that apply._

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 **/target*
 **/*.rs.bk
+/contrib/**/.vscode
```
Comment (imeoer) [inline .gitignore:3]: `**/.vscode` should be better ?

Comment (zyfjeff) [reply .gitignore]: fixed

**Review by imeoer — APPROVED** [review]

**Comment by adamqqqplay** [comment]
Need squash.

**Pull request merged**